docs/use_cases/recommender_from_clickstreams.md

```diff
@@ -0,0 +1,212 @@
+| Azure Sands Hotel       | Summit Vista Inn        | Yes       |
+| Azure Sands Hotel       | Evergreen Retreat Lodge | Yes       |
+| Evergreen Retreat Lodge | Azure Sands Hotel       | Yes       |
+
+Do you see the problem when building a binary classifier for this table? The target column *Neighbor?* only contains a single label since this is how we have constructed this table! So, what the authors proposed was to use **negative sampling** to save the day.
+
+### Negative sampling
+The idea is simple: 
+>If you only have *positive* rows consisting of neighbors, just make up some random pairs of hotels that were not seen next to each other in a sequence.
+This can look like this:
+
+| Focus                   | Other Hotel             | Neighbor? |
+|-------------------------|-------------------------|-----------|
+| Tranquil Haven Resort   | Summit Vista Inn        | Yes       |
+| Tranquil Haven Resort   | Some Other Hotel 1      | No        |
+| Summit Vista Inn        | Tranquil Haven Resort   | Yes       |
+| Summit Vista Inn        | Some Other Hotel 2      | No        |
+| Summit Vista Inn        | Azure Sands Hotel       | Yes       |
+| Summit Vista Inn        | Some Other Hotel 1      | No        |
+| Azure Sands Hotel       | Summit Vista Inn        | Yes       |
+| Azure Sands Hotel       | Some Other Hotel 3      | No        |
+| Azure Sands Hotel       | Evergreen Retreat Lodge | Yes       |
+| Azure Sands Hotel       | Some Other Hotel 3      | No        |
+| Evergreen Retreat Lodge | Azure Sands Hotel       | Yes       |
+| Evergreen Retreat Lodge | Some Other Hotel 1      | No        |
+
+And now, we can build a binary classifier that has to solve the reasonable classification task of finding out whether two hotels were clicked successively, or not.
+
+And this is what happens under the hood in a nutshell. Of course, there are many more hyperparameters that you can tune to get better embeddings. We have seen the embedding size (called `vector_size` in Gensim) already. Another important hyperparameter is **window size** using the `window_size` parameter. This enlarges the focus of the model, so we are not only interested in the direct left and right neighbors of the hotel but also hotels that are maybe two or three steps apart from the focus element.
+
+## Conclusion
+We have seen how easy it is to get embeddings for hotels - or rather any type of item - that users click on in an online session. We only have to record the clickstream data and preprocess it a bit, so we can use Gensim to train a model in a few lines of code.
+
+It is probably one of the easiest projects to set up, and still, you get really interesting results from it in the form of embeddings. You can then use these embeddings to either build a recommender system, but you can also just visualize them to gain more insights into your data.
+
+As an example, you can use PCA, t-SNE, or UMAP to project the embeddings into two dimensions like this:
```

Comment:
I would suggest to use UMAP as I believe it is widely considered superior to T-SNE, for example see this [resource](https://pair-code.github.io/understanding-umap/)

Reply:
Heya! Thanks a lot for the feedback :) yes, I wish think that UMAP is better, I just wanted to work with the libraries that most people have installed already. But I'll change it to UMAP then later!